ts/features/idpay/initiative/timeline/components/TimelineTransactionDetailsComponent.tsx

```diff
@@ -47,6 +47,18 @@ const TimelineTransactionDetailsComponent = (props: Props) => {
     O.toNullable
   );
 
+  const idTrxIssuer = transaction.idTrxIssuer || "";
+  const idTrxAcquirer = transaction.idTrxAcquirer || "";
+
+  const formattedAmount = pipe(
+    transaction.amount,
+    O.fromNullable,
+    O.map(amount => formatNumberAmount(amount, true)),
+    O.getOrElse(() => "-")
+  );
```

Comment:
what about a `formatNumberCurrencyOrDefaultRightSign` function? or maybe a small refactor of the already existing function in order to pass an optional boolean parameter

Reply:
It's only used here, IMHO there is no need to generalize its usage for now.